graylog2_handler/__init__.py

```diff
@@ -0,0 +1,32 @@
+import openerp.tools as tools
+import logging
+
+_logger = logging.getLogger(__name__)
+
+# Check if this module is enabled in config
+if tools.config.get('gelf_enabled', False):
+    try:
+        import graypy
+    except ImportError:
+        _logger.error("Package 'graypy' not installed")
+    else:
```

Comment:
Maybe a nitpick, but do we really need for the next code to be in an `else` block?